veripy/steps/content/checks/element_visible.py

```diff
@@ -31,3 +23,36 @@ def check_element_visible(context, element_name, not_):
             raise AssertionError(
                 f'The "{element_name}" was supposed to be visible but it was not present.'
             )
+
+
+@then('the following {colname} are {not_:optional_not}visible')
+def check_elements_visible(context, colname, not_):
+    """ Asserts that the element is the visible on the page.
+    """
+    if getattr(context, 'table') is None:
+        raise AssertionError('No items were defined.')
+    if not context.table.has_column(colname):
+        raise AssertionError(f'The table does not have a header \'{colname}\'.')
```

Comment:
> When a string contains single or double quote characters, however, use the other one to avoid backslashes in the string. It improves readability.

* https://pep8.org/#string-quotes